src/all/kavita/src/eu/kanade/tachiyomi/extension/all/kavita/Kavita.kt

```diff
@@ -401,8 +306,17 @@ class Kavita(private val suffix: String = "") : ConfigurableSource, UnmeteredSou
             sortOptions = SortOptions(SortFieldEnum.AverageRating.type, false),
             statements = mutableListOf(),
         )
-        val payload = json.encodeToJsonElement(filter).toString()
 
+        // Always exclude EPUBs unless explicitly included
+        if (!preferences.getBoolean(SHOW_EPUB_PREF, SHOW_EPUB_DEFAULT)) {
+            filter.addStatement(
+                FilterComparison.NotContains,
+                FilterField.Formats,
+                "3",
```

Comment:
Avoid magic numbers/strings as much as possible. I would put the Formats as an enum.